pkg/ccl/changefeedccl/cdcevent/event.go

```diff
@@ -496,6 +498,31 @@ const (
 // DecodeKV decodes key value at specified schema timestamp.
 func (d *eventDecoder) DecodeKV(
 	ctx context.Context, kv roachpb.KeyValue, rt RowType, schemaTS hlc.Timestamp, keyOnly bool,
+) (Row, error) {
+	r, err := d.decodeKV(ctx, kv, rt, schemaTS, keyOnly)
+	if err == nil {
+		return r, nil
+	}
+
+	// Failure to decode roachpb.KeyValue we received from rangefeed is pretty bad.
+	// At this point, we only have guesses why this happened (schema change? data corruption?).
+	// Retrying this error however is likely to produce exactly the same result.
+	// So, be loud and treat this error as a terminal changefeed error.
+	kvBytes, marshalErr := protoutil.Marshal(&kv)
+	if marshalErr != nil {
+		// That's mighty surprising.  Just shove error message into kvBytes.
+		kvBytes = []byte(fmt.Sprintf("marshalErr<%s>", marshalErr.Error()))
```

Comment:
ewww? why is this going in the bytes, that have nothing to do with it? why not CombineErrors or Wrapf?

Reply:
I could have wrapped another error here... true.. Just wans't done.. cause it's really "surprising" and I didn't care to repeat logging/wrap...